pkg/controller/shoot/shoot.go

```diff
@@ -167,6 +169,25 @@ func (c *Controller) Run(shootWorkers, shootCareWorkers, shootMaintenanceWorkers
 	}
 	logger.Logger.Info("Shoot controller initialized.")
 
+	// Get all shoots managed by Gardener and check if the backup defaults are valid. If not, update the shoots
+	// with the default backup schedule.
+	shoots, err := c.shootLister.List(labels.Everything())
+	if err != nil {
+		logger.Logger.Errorf("Failed to fetch shoots resources: %v", err.Error())
+		return
+	}
+	for _, shoot := range shoots {
+		if shoot.Spec.Backup.Schedule != garden.DefaultETCDBackupSchedule {
```

Comment:
Hm, if I would want to configure another schedule (twice a day for example) my change would get overridden every time the Gardener controller manager restarts.  Instead of checking whether `shoot.Spec.Backup.Schedule` is not the default value could we rather check whether the value is smaller than the lowest frequency we support (and only update in those cases)?